__tests__/api.test.js

```diff
@@ -0,0 +1,39 @@
+const app = require(`${__dirname}/../app.js`);
+const request = require("supertest");
+const db = require(`${__dirname}/../db`);
+const data = require(`${__dirname}/../db/data`);
+const seed = require(`${__dirname}/../db/seeds/seed.js`);
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/categories", () => {
+  test("Returns an array", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: { categories } }) => {
+        expect(categories).toBeInstanceOf(Array);
+      });
+  });
+  test("Returns a status of 200 and all categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: { categories } }) => {
+        expect(categories).toHaveLength(4);
+      });
+  });
```

Comment:
Might be worth making an assertion on what the category object look like, I think it would be fine to do this within this test